src/pysdmx/model/submission.py

```diff
@@ -0,0 +1,20 @@
+"""SDMX Submission classes."""
```

Comment:
This class could also go to the message module, IMHO.

Reply:
No strong opinion on why it should be on a separate file, so moving it.